src/main/java/uk/gov/hmcts/juror/api/juror/service/JurorServiceImpl.java

```diff
@@ -263,7 +269,7 @@ public DigitalResponse convertJurorResponseDtoToEntity(JurorResponseDto dto) {
             );
         }
 
-        final JurorPool jurorDetails = jurorRepository.findByJurorJurorNumber(jurorNumber);
+        final JurorPool jurorDetails = jurorPoolRepository.findByJurorJurorNumber(jurorNumber);
```

Comment:
See above

Reply:
this is just a rename from jurorRepository to jurorPoolRepository